src/regression/poly-fit-regression2d.js

```diff
@@ -66,29 +66,19 @@ export default class PolynomialFitRegression2D extends BaseRegression {
 
     var examples = X.rows;
     var coefficients = ((this.order + 2) * (this.order + 1)) / 2;
+    if (examples < coefficients) {
+      throw new Error(
+        'Insufficient number of points to create regression model.',
```

Comment:
Would give the minimum number of expected points for the model in the error message.

Reply:
It depends how the points are independent so it depends. This is why we didn't put the number.